modules/clientnotify.cpp

```diff
@@ -76,18 +89,37 @@ class CClientNotifyMod : public CModule {
 
     void OnClientLogin() override {
         CString sRemoteIP = GetClient()->GetRemoteIP();
-        if (!m_bNewOnly ||
-            m_sClientsSeen.find(sRemoteIP) == m_sClientsSeen.end()) {
+        CString sRemoteClientID = GetClient()->GetIdentifier();
+
+        CString& sClientNameMessage = sRemoteIP;
+        if (m_bNotifyOnNewClientID and sRemoteClientID != "") {
+            sClientNameMessage = sRemoteClientID;
```

Comment:
I think it would be more useful to show IP always, and id if client sent one. Now id would hide the ip